earth_enterprise/docker/image-definition/Dockerfile.build-env.rhel-7.template

```diff
@@ -0,0 +1,26 @@
+# We only need a few context files, rather than the whole repository:
+# --[ TST-Dockerfile: context-dir=earth_enterprise/docker/image-definition ]--
+# Prompt the user to enter variables required to build this Dockerfile:
+# --[ TST-Dockerfile: required-vars=REDHAT_USERNAME ]--
+# Avoid displaying the values of some variables:
+# --[ TST-Dockerfile: required-password-vars=REDHAT_PASSWORD ]--
+
+FROM registry.access.redhat.com/rhel:7.3
```

Comment:
7.4 is out now. Maybe we should consider just using :latest

Reply:
Or at least updating to 7.4

Reply:
I could update it.  The problem with using `latest` is that it'll become RHEL 8 at some time.